requirements.txt

```diff
@@ -40,7 +40,7 @@ tiktoken>=0.9.0
 transformers>=4.52.4
 transformer-engine[jax]>=2.1.0
 jsonlines>=4.0.0
-pathwaysutils>=0.1.1
+pathwaysutils@git+https://github.com/AI-Hypercomputer/pathways-utils.git
```

Comment:
Is this also needed in these files?

* `constraints_gpu.txt`
* `requirements_with_jax_ai_image.txt`
* `requirements_with_jax_stable_stack_0_6_1_pipreqs.txt`